[2023-07-27T09:49:57.592+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_engineer_jobs_dag.extract_relevant_records_from_overall_data scheduled__2023-04-02T00:00:00+00:00 [queued]>
[2023-07-27T09:49:57.721+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_engineer_jobs_dag.extract_relevant_records_from_overall_data scheduled__2023-04-02T00:00:00+00:00 [queued]>
[2023-07-27T09:49:57.723+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-07-27T09:49:57.754+0000] {taskinstance.py:1284} INFO - Starting attempt 2 of 2
[2023-07-27T09:49:57.757+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-07-27T09:49:58.029+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract_relevant_records_from_overall_data> on 2023-04-02 00:00:00+00:00
[2023-07-27T09:49:58.047+0000] {standard_task_runner.py:55} INFO - Started process 10887 to run task
[2023-07-27T09:49:58.143+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_engineer_jobs_dag', 'extract_relevant_records_from_overall_data', 'scheduled__2023-04-02T00:00:00+00:00', '--job-id', '1466', '--raw', '--subdir', 'DAGS_FOLDER/load_data.py', '--cfg-path', '/tmp/tmpq4jpm7wt']
[2023-07-27T09:49:58.163+0000] {standard_task_runner.py:83} INFO - Job 1466: Subtask extract_relevant_records_from_overall_data
[2023-07-27T09:49:58.862+0000] {task_command.py:389} INFO - Running <TaskInstance: data_engineer_jobs_dag.extract_relevant_records_from_overall_data scheduled__2023-04-02T00:00:00+00:00 [running]> on host bc3380e7707e
[2023-07-27T09:50:00.040+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=kasimir.Ikuenobe@hotmail.com
AIRFLOW_CTX_DAG_OWNER=ebejale
AIRFLOW_CTX_DAG_ID=data_engineer_jobs_dag
AIRFLOW_CTX_TASK_ID=extract_relevant_records_from_overall_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-02T00:00:00+00:00
[2023-07-27T09:50:00.177+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/load_data.py", line 81, in extract_relevant_records_from_overall_data
    for i in range(len(data['data'])):
KeyError: 'data'
[2023-07-27T09:50:00.299+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=data_engineer_jobs_dag, task_id=extract_relevant_records_from_overall_data, execution_date=20230402T000000, start_date=20230727T094957, end_date=20230727T095000
[2023-07-27T09:50:00.673+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 1466 for task extract_relevant_records_from_overall_data ('data'; 10887)
[2023-07-27T09:50:00.828+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-07-27T09:50:02.790+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
